rustls/src/client/client_conn.rs

```diff
@@ -774,6 +774,11 @@ mod connection {
             self.inner.core.data.ech_status
         }
 
+        /// Returns the number of TLS1.3 tickets that have been received.
+        pub fn tls13_tickets_received(&self) -> u32 {
```

Comment:
I think the `UnbufferedClientConnection` needs a `pub` accessor like this as well.

Reply:
Thanks!